src/animation/AnimationClip.js

```diff
@@ -420,6 +420,45 @@ Object.assign( AnimationClip.prototype, {
 		return this;
 
 	},
+	
+	slice: function(from, to, name) {
+
+            var frames = to - from, duration = 0;
+
+            var tracks = [];
+
+            for ( var i = 0, l = this.tracks.length; i < l; i ++ ) {
+
+                var track = this.tracks[i];
```

Comment:
Code style: Extra spaces are necessary after opening bracket and before closing bracket.

https://github.com/mrdoob/three.js/wiki/Mr.doob's-Code-Style%E2%84%A2#arrays